test_conformance/api/test_api_min_max.cpp

```diff
@@ -1575,9 +1542,8 @@ int test_min_max_constant_buffer_size(cl_device_id deviceID, cl_context context,
     MTdata d;
 
     /* Verify our test buffer won't be bigger than allowed */
-    error = clGetDeviceInfo(deviceID, CL_DEVICE_MAX_CONSTANT_BUFFER_SIZE,
-                            sizeof(maxSize), &maxSize, 0);
-    test_error(error, "Unable to get max constant buffer size");
+    maxSize = get_device_info_max_constant_buffer_size(
+        deviceID, MAX_DEVICE_MEMORY_SIZE_DIVISOR);
 
     if ((0 == gIsEmbedded && maxSize < 64L * 1024L) || maxSize < 1L * 1024L)
```

Comment:
Looks like this should be 
+    if ((0 == gIsEmbedded && maxSize*MAX_DEVICE_MEMORY_SIZE_DIVISOR < 64L * 1024L) || maxSize*MAX_DEVICE_MEMORY_SIZE_DIVISOR < 1L * 1024L)

Reply:
Done.